### ISSUE TITLE
Investigate DartPad behavior for accessibility: zoomed-in windows

### ISSUE THREAD

**Issue by legalcodes** [dart-lang/dart-pad#1340]
@sfshaza2 helped find some cases where DartPad doesn't behave as expected with a chrome window that is reasonably zoomed-in for accessibility purposes. 

@sfshaza2 any chance you can provide some screenshots?

**Comment by RedBrogdon** [comment]
If this is going to stay open, we need something actionable. @legalcodes/Shams, can you help define the failure cases so they can be fixed?

**Comment by sfshaza2** [comment]
Oh wow, I can't even remember how to make this happen. Go ahead and close it.
